lib/rollbax/item.ex

```diff
@@ -2,33 +2,76 @@ defmodule Rollbax.Item do
   def draft(token, envt) do
     {:ok, host} = :inet.gethostname
     %{
-      "access_token" => token,
-      "data" => %{
-        "server" => %{
-          "host" => List.to_string(host)
+      access_token: token,
+      data: %{
+        server: %{
+          host: List.to_string(host)
         },
-        "environment" => envt,
-        "language" => language(),
-        "platform" => platform(),
-        "notifier" => notifier()
+        environment: envt,
+        language: language(),
+        platform: platform(),
+        notifier: notifier()
       }
     }
   end
 
-  def compose(draft, {level, msg, time, meta}) do
+  def compose(draft, {level, e, stacktrace, time, meta}) do
```

Comment:
Let's call `e` `exception` instead (as you did in other places). This applies to a bunch of other cases below :)